internal/auth/oidc/account.go

```diff
@@ -118,14 +118,14 @@ func (a *Account) SetTableName(n string) {
 }
 
 // oplog will create oplog metadata for the Account.
-func (c *Account) oplog(op oplog.OpType, authMethodScopeId string) oplog.Metadata {
+func (c *Account) oplog(op oplog.OpType) oplog.Metadata {
```

Comment:
why have you removed scope id?  This is an issue that we're going to have to fix across all the resources that don't have it.  Please add scope id back into the metadata.